### DISCUSSION TITLE
Green Software Speaker page is broken

### DISCUSSION THREAD

**Discussion by fullo** [Green-Software-Foundation/discussions#127] [Community Working Group]
The Green Software Speaker's link/widget in the [GSF's projects page](https://greensoftware.foundation/projects)  is broken, the link should bring visitors to the [Champion page](https://champions.greensoftware.foundation/champions/)

**Comment by russelltrow** [comment]
Hi @fullo thank you so much for spotting this. I will get it raised internally.
